apis/core/v1beta1/featureflagsource_types.go

```diff
@@ -172,3 +204,193 @@ func (s SyncProviderType) IsGrpc() bool {
 func (s SyncProviderType) IsFlagdProxy() bool {
 	return s == SyncProviderFlagdProxy
 }
+
+//nolint:gocyclo
+func NewFeatureFlagSourceSpec() (*FeatureFlagSourceSpec, error) {
+	fsc := &FeatureFlagSourceSpec{
+		MetricsPort:         DefaultMetricPort,
```

Comment:
Note - this is good time to change this to align flagd change https://github.com/open-feature/flagd/pull/1012

Reply:
:100: